### ISSUE TITLE
CPTPLND conversion bug for depolarized CPHASE gate

### ISSUE THREAD

**Issue by kmrudin** [pyGSTio/pyGSTi#295]
**Describe the bug**
Attempting to convert a model with depolarizing error on Gcphase from dense parameterization to CPTPLND fails.

**To Reproduce**
The following is a minimal working example:

```import pygsti
from pygsti.modelpacks import smq2Q_XYCPHASE

noisy_model = smq2Q_XYCPHASE.target_model()
noisy_model['Gcphase',0,1].depolarize(0.001)
noisy_model.set_all_parameterizations('CPTPLND')
```

This yields the following warning and error:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/var/folders/4m/84t3c8fx3x39j2bj2srdb4d8002m7b/T/ipykernel_32267/2949875122.py in <module>
      1 noisy_model = smq2Q_XYCPHASE.target_model()
      2 noisy_model['Gcphase',0,1].depolarize(0.001)
----> 3 noisy_model.set_all_parameterizations('CPTPLND')

~/pyGSTi-0.9.10-develop/pygsti/models/explicitmodel.py in set_all_parameterizations(self, gate_type, prep_type, povm_type, instrument_type, extra)
    430         ityp = _instrument.instrument_type_from_op_type(gate_type) if instrument_type == "auto" else instrument_type
    431 
--> 432         self.convert_members_inplace(typ, 'operations', 'all', flatten_structure=True)
    433         self.convert_members_inplace(ityp, 'instruments', 'all', flatten_structure=True)
    434         self.convert_members_inplace(rtyp, 'preps', 'all', flatten_structure=True)

~/pyGSTi-0.9.10-develop/pygsti/models/explicitmodel.py in convert_members_inplace(self, to_type, categories_to_convert, labels_to_convert, ideal_model, flatten_structure, set_default_gauge_group)
    349                 if labels_to_convert == 'all' or lbl in labels_to_convert:
    350                     ideal = ideal_model.operations.get(lbl, None) if (ideal_model is not None) else None
--> 351                     self.operations[lbl] = _op.convert(gate, to_type, self.basis, ideal, flatten_structure)
    352         if any([c in categories_to_convert for c in ('all', 'instruments')]):
    353             for lbl, inst in self.instruments.items():

~/pyGSTi-0.9.10-develop/pygsti/modelmembers/operations/__init__.py in convert(operation, to_type, basis, ideal_operation, flatten_structure)
    372             error_msgs[to_type] = str(e)  # try next to_type
    373 
--> 374     raise ValueError("Could not convert operation to to type(s): %s\n%s" % (str(to_types), str(error_msgs)))
    375 
    376 

ValueError: Could not convert operation to to type(s): ('CPTPLND',)
{'CPTPLND': 'Could not create an operator of type(s) CPTPLND from the given superop:\n{\'CPTPLND\': "Failure to create CPTP operation (maybe due the complex log\'s branch cut?)"}'}
```

Note that this does _not_ fail when the depolarized gate is a local gate (e.g., Gxpi2:0).

**Expected behavior**
The parameterization of the model should change to CPTPLND.

**Environment**
 - pyGSTi 0.9.10.1.post175 (develop branch)
 - python 3.8.13
 - macOS 12.6.3

**Comment by sserita** [comment]
As the error message states, this was likely due to branch cut issues with the CPHASE gate (since it has two -1 eigenvalues). This can be fixed by passing in the target model to `set_all_parameterizations` (which is a new feature as of this bugfix).

A minimal working example now would be as follows:
```
from pygsti.modelpacks import smq2Q_XYCPHASE

target_model = smq2Q_XYCPHASE.target_model()
noisy_model = target_model.copy()
noisy_model['Gcphase',0,1].depolarize(0.001)
noisy_model.set_all_parameterizations('CPTPLND', ideal_model=target_model)
```

This change also include an updated error message. Including just the last few lines of the error message:
```
ValueError: Could not convert operation to to type(s): ('CPTPLND',)
{'CPTPLND': 'Could not create an operator of type(s) CPTPLND from the given superop:\n{\'CPTPLND\': "Failure to create CPTP operation (maybe due the complex log\'s branch cut?)"}'}

The above exception was the direct cause of the following exception:

ValueError                                Traceback (most recent call last)
Cell In[2], line 6
      4 noisy_model = target_model.copy()
      5 noisy_model['Gcphase',0,1].depolarize(0.001)
----> 6 noisy_model.set_all_parameterizations('CPTPLND')

File ~/Documents/repos/pyGSTi/pygsti/models/explicitmodel.py:441, in ExplicitOpModel.set_all_parameterizations(self, gate_type, prep_type, povm_type, instrument_type, ideal_model)
    439     self.convert_members_inplace(povmtyp, 'povms', 'all', flatten_structure=True, ideal_model=static_model)
    440 except ValueError as e:
--> 441     raise ValueError("Failed to convert members. If converting to CPTP-based models, " +
    442         "try providing a target model to avoid possible branch cuts.") from e
    444 self.set_default_gauge_group_for_member_type(typ)

ValueError: Failed to convert members. If converting to CPTP-based models, try providing an ideal_model to avoid possible branch cuts.
```

**Comment by sserita** [comment]
Closed as this is now in 0.9.11 release
